pkg/services/ngalert/api/tooling/definitions/upgrade.go

```diff
@@ -0,0 +1,189 @@
+package definitions
+
+// swagger:route GET /api/v1/upgrade/org upgrade RouteGetOrgUpgrade
+//
+// Get existing alerting upgrade for the current organization.
+//
+//     Produces:
+//     - application/json
+//
+//     Responses:
+//       200: OrgMigrationState
+
+// swagger:route POST /api/v1/upgrade/org upgrade RoutePostUpgradeOrg
+//
+// Upgrade all legacy alerts for the current organization.
+//
+//     Produces:
+//     - application/json
+//
+//     Responses:
+//       200: OrgMigrationSummary
+
+// swagger:route DELETE /api/v1/upgrade/org upgrade RouteDeleteOrgUpgrade
+//
+// Delete existing alerting upgrade for the current organization.
+//
+//     Produces:
+//     - application/json
+//
+//     Responses:
+//       200: Ack
+
+// swagger:route POST /api/v1/upgrade/dashboards/{DashboardID}/panels/{PanelID} upgrade RoutePostUpgradeAlert
+//
+// Upgrade single legacy dashboard alert for the current organization.
+//
+//     Produces:
+//     - application/json
+//
+//     Responses:
+//       200: OrgMigrationSummary
+
+// swagger:route POST /api/v1/upgrade/dashboards/{DashboardID} upgrade RoutePostUpgradeDashboard
+//
+// Upgrade all legacy dashboard alerts on a dashboard for the current organization.
+//
+//     Produces:
+//     - application/json
+//
+//     Responses:
+//       200: OrgMigrationSummary
+
+// swagger:route POST /api/v1/upgrade/dashboards upgrade RoutePostUpgradeAllDashboards
+//
+// Upgrade all legacy dashboard alerts for the current organization.
+//
+//     Produces:
+//     - application/json
+//
+//     Responses:
+//       200: OrgMigrationSummary
+
+// swagger:route POST /api/v1/upgrade/channels upgrade RoutePostUpgradeAllChannels
+//
+// Upgrade all legacy notification channels for the current organization.
+//
+//     Produces:
+//     - application/json
+//
+//     Responses:
+//       200: OrgMigrationSummary
+
+// swagger:route POST /api/v1/upgrade/channels/{ChannelID} upgrade RoutePostUpgradeChannel
+//
+// Upgrade a single legacy notification channel for the current organization.
+//
+//     Produces:
+//     - application/json
+//
+//     Responses:
+//       200: OrgMigrationSummary
```

Comment:
The definitions above do not reflect all possible responses such as 409 and 400 etc. I do not insist on fixing this because we generally miss them all over APIs